concrete/src/Error/ErrorList/Error/Error.php

```diff
@@ -1,16 +1,23 @@
 <?php
+
 namespace Concrete\Core\Error\ErrorList\Error;
 
 use Concrete\Core\Error\ErrorList\Field\FieldInterface;
 
 class Error extends AbstractError
 {
-
+    /**
+     * The error message.
+     *
+     * @var string
```

Comment:
We don't enforce type currently, do we? https://github.com/concrete5/concrete5/pull/7375/files#diff-d9fd61b2b85566c1b3dd97d31cd594d6R24

Reply:
What about this type hint?

```
string|mixed a string, a scalar, or an object that implements the __toString() method
```

Reply:
Or `(string) $message` in the constructor?